mappings/net/minecraft/entity/passive/VillagerEntity.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS aua net/minecraft/entity/passive/VillagerEntity
 	FIELD bD VILLAGER_DATA Lqe;
+	METHOD a initBrainSchedule (Laiu;)V
```

Comment:
I think this is responsible for general brain initialization, not just the schedule